api/types/database.go

```diff
@@ -680,6 +693,16 @@ func (gcp GCPCloudSQL) GetServerName() string {
 	return fmt.Sprintf("%s:%s", gcp.ProjectID, gcp.InstanceID)
 }
 
+// HasValidID returns true if any of the identifiers exists.
+func (rds RDS) HasValidID() bool {
```

Comment:
```suggestion
func (rds RDS) IsValid() bool {
```
HasValidID at first glance suggest that RDS struct has a ID field that we are validating in the `HasValidID` call.